### DISCUSSION TITLE
Frigate+ website dashboard: "Images"

### DISCUSSION THREAD

**Discussion by detroitgoose** [blakeblackshear/frigate#10710] [Ideas]
Hello and thank you for Frigate+! I've experienced immediate improvements in detection.

Image annotation itself is easy enough, and I like how it automatically goes to the next unverified image in the list, but searching through the thumbnails of "saved and verified" images is a major pain. For instance, I realized far too late that when I get a false positive, and mark it as "not a cat," I should have been annotating that it was in fact a dog. But, going back and finding those false positive images is incredibly difficult. One of the issues is that if you view an already verified image and then "go back" to the main thumbnail page, you're put back at the top of the list and have to scroll down to find the next image. Just being able to go to the next or previous image from the "edit image/annotate" page would be _extremely_ helpful. I imagine this could be easily implemented using left and right arrows on either side of the edit page, as well as by using the keyboard arrows.

I'd also like to suggest the ability to sort the thumbnails by cameras, labels, and false positives. I could see this being identical to how you can sort events through the NVR by ticking the label/camera boxes. 

The website in its current state is rather nice for the initial annotation, but nearly impossible to efficiently revise previously verified images in an efficient manner once the image list grows to many hundreds/thousands.

**Comment by detroitgoose** [comment]
I see now that this general suggestion has already been covered and should have been in the "Frigate+ Model Launch" discussion

**Reply by heisenberg2980** [reply]
Having the same issue, can you share the link where this is covered?

**Reply by detroitgoose** [reply]
@heisenberg2980  sorry late response. It's a very long discussion. the TL;DR is that it's planned for future updates. for now it is what it is

https://github.com/blakeblackshear/frigate/discussions/7932
